app/assets/javascripts/app/components/button/button.js

```diff
@@ -27,7 +27,10 @@
   function councilRaisedButton() {
     return {
       restrict: 'E',
-      scope: {},
+      scope: {
+        disabled: '=',
+        click: '&'
+      },
```

Comment:
I'm not a big fan of exceptions, it makes sense now for the moment, for the people looking at it with a context. In another time, when someone else sees this, he might not know the context and believe it's the norm.
Anyway, I don't believe that in this component's case, it's really necessary due to the use you're giving to the properties. Plus, the component, being isolated by nature, won't have any conflicts with other outer scopes. One more plus, the problem with "using $scope" is actually using it explicitly, in this case you're relying on a scope (notice the "a", i.e. a conceptual scope), which will still exist in angular 2, so binding things without a namespace is ok in this circumstances.